stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/viewholder/attachment/AttachmentFactories.kt

```diff
@@ -0,0 +1,40 @@
+package io.getstream.chat.android.ui.message.list.adapter.viewholder.attachment
+
+import android.view.ViewGroup
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.ui.message.list.adapter.MessageListListenerContainer
+
+/**
+ * A manager for registered custom attachment factories.
+ */
+public class AttachmentFactories(
```

Comment:
Maybe `AttachmentFactory(Factories)Manager`?

Reply:
Yes, agree, no need to make this the "Compose" way. Renamed to `AttachmentFactoryManager`.